source/_patterns/00-atoms/components/social-media-sharers.mustache

```diff
@@ -0,0 +1,38 @@
+<div class="social-media-sharers">
+
+  {{! Skeleton autogenerated by http://sharingbuttons.io/
+      See notes in social-media-sharer.scss for more details. }}
+
+  <!-- Sharingbutton Facebook -->
+  <a class="social-media-sharer" href="{{{facebookUrl}}}" target="_blank" aria-label="Share on Facebook">
+    <div class="social-media-sharer__icon_wrapper social-media-sharer__icon_wrapper--facebook social-media-sharer__icon_wrapper--small"><div aria-hidden="true" class="social-media-sharer__icon social-media-sharer__icon--solid">
+      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18.77 7.46H14.5v-1.9c0-.9.6-1.1 1-1.1h3V.5h-4.33C10.24.5 9.5 3.44 9.5 5.32v2.15h-3v4h3v12h5v-12h3.85l.42-4z"/></svg>
+    </div>
+    </div>
+  </a>
+
+  <!-- Sharingbutton Twitter -->
+  <a class="social-media-sharer" href="{{{twitterUrl}}}" target="_blank" aria-label="Tweet a link to this page">
```

Comment:
https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

Reply:
Good spot.